opni-insights-service/app/main.py

```diff
@@ -307,3 +372,29 @@ async def index_workload(start_ts: int, end_ts: int):
     except Exception as e:
         # Bad Request
         logging.error(e)
+
+
+@app.get("/overall_insights")
+async def index_overall_breakdown(start_ts: int, end_ts: int):
+    # This function handles get requests for fetching workload breakdown insights.
+    logging.info(
+        f"Received request to obtain all insights between {start_ts} and {end_ts}"
+    )
+    try:
+        result = get_overall_breakdown(start_ts, end_ts)
+        return result
+    except Exception as e:
+        # Bad Request
+        logging.error(e)
+
+
+@app.get("/logs")
+async def index_logs(start_ts: int, end_ts: int):
+    # This function handles get requests for fetching workload breakdown insights.
+    logging.info(f"Received request to obtain all logs between {start_ts} and {end_ts}")
```

Comment:
the logging here says it requests for all logs, however this function `get_logs` returns non-normal logs?

Reply:
@AmartC ^